lite/examples/model_personalization/converter/tfltransfer/heads/keras_model_head.py

```diff
@@ -143,12 +148,16 @@ def train(self, bottleneck, labels, scope='head'):
         name=scope,
         input_map=input_map,
         return_elements=[loss_name])[0]
+    train_variables = [
+        tfv1.get_default_graph().get_tensor_by_name(scope + '/' + name)
+        for name in self._trainable_variable_names
+    ]
     variables = [
         tfv1.get_default_graph().get_tensor_by_name(scope + '/' + name)
         for name in self._variable_names
     ]
     with tf.name_scope(scope + '/backprop'):
-      gradients = tf.gradients(loss, variables, stop_gradients=variables)
+      gradients = tf.gradients(loss, train_variables, stop_gradients=train_variables)
```

Comment:
Could you describe & add the example of the discrepancy between variables and train_variables?

In this case, you don't use variables, then it should be replaced at the return (e.g. return loss, gradients, train_variables) and remove all things about variables.

Could you check it?